lib/diagnostic.rb

```diff
@@ -100,15 +115,18 @@ def eat_meat(food)
 end
 
 ##
-# your response here
-##
+Class Lion < Cat
+  def roar
+    puts 'ROAR!'
+  end
+  ##
 
 # #
 # What are some of the advantages of using composition (i.e. mixins)
 # over using direct inheritance?
 
 ##
-# your response as a comment here
+# They can be very specific to the already specific instance of the Class
```

Comment:
In direct inheritance, you're limited to inheriting from a single source. In composition, you can draw from as many modules as you like.